似乎是在更新2.16.0-RC后启动报错

`
[root@VM-0-3-centos mcl]# ./mcl
15:39:20 [INFO] iTX Technologies Mirai Console Loader version 2.1.2-61c8bd8
15:39:20 [INFO] Runtime: OpenJDK 64-Bit Server VM 17.0.6 (arch: 64)
15:39:20 [INFO] https://github.com/iTXTech/mirai-console-loader
15:39:20 [INFO] This program is licensed under GNU AGPL v3
15:39:21 [INFO] Mirai Console Loader Announcement:
Mirai Console Loader 公告栏

如果在图片上传的时候遇到问题请与我们联系 (需要提供图片文件源本)
- 如 Unsupported image type for ExternalResource * considering use gif/png/bmp/jpg format.
`- Tracker: https://github.com/mamoe/mirai/issues/new/choose

常用资源整合
`- https://mirai.mamoe.net/topic/653

MCL 已推出 2.1.0，更好的支持 Mirai 2.11 的插件系统，建议更新。

15:39:21 [INFO] Verifying "net.mamoe:mirai-console" v2.16.0-RC
15:39:21 [INFO] Verifying "net.mamoe:mirai-console-terminal" v2.16.0-RC
15:39:22 [INFO] Verifying "net.mamoe:mirai-core-all" v2.16.0-RC
15:39:22 [INFO] Verifying "org.itxtech:mcl-addon" v2.1.1
15:39:22 [INFO] Verifying "net.mamoe:chat-command" v0.5.1
15:39:22 [INFO] Verifying "net.mamoe:mirai-api-http" v2.9.1
15:39:22 [WARN] Package "net.mamoe:chat-command" requires "net.mamoe:mirai-console" version 2.0-RC. Current version is 2.16.0-RC
2023-09-04 15:39:23 I/main: Starting mirai-console...
2023-09-04 15:39:23 I/main:

=================================[ Mirai console 2.16.0-RC ]==================================

| \ / \ \ | / \ |
| ▓▓\ / ▓▓\▓▓ ______ ______ \▓▓ ▓▓▓▓▓▓\ ______ _______ _______ ______ | ▓▓ ______
| ▓▓▓\ / ▓▓▓ / \ | | \ ▓▓ \▓▓/ | \ / / | ▓▓/
| ▓▓▓▓\ ▓▓▓▓ ▓▓ ▓▓▓▓▓▓\ \▓▓▓▓▓▓\ ▓▓ ▓▓ | ▓▓▓▓▓▓\ ▓▓▓▓▓▓▓\ ▓▓▓▓▓▓▓ ▓▓▓▓▓▓\ ▓▓ ▓▓▓▓▓▓
| ▓▓\▓▓ ▓▓ ▓▓ ▓▓ ▓▓ \▓▓/ ▓▓ ▓▓ ▓▓ | ▓▓ | ▓▓ ▓▓ | ▓▓\▓▓ | ▓▓ | ▓▓ ▓▓ ▓▓ ▓▓
| ▓▓ \▓▓▓| ▓▓ ▓▓ ▓▓ | ▓▓▓▓▓▓▓ ▓▓ ▓▓/ \ ▓▓__/ ▓▓ ▓▓ | ▓▓_\▓▓▓▓▓▓\ ▓▓__/ ▓▓ ▓▓ ▓▓▓▓▓▓▓▓
| ▓▓ \▓ | ▓▓ ▓▓ ▓▓ \▓▓ ▓▓ ▓▓\▓▓ ▓▓\▓▓ ▓▓ ▓▓ | ▓▓ ▓▓\▓▓ ▓▓ ▓▓\▓▓
\▓▓ \▓▓\▓▓\▓▓ \▓▓▓▓▓▓▓\▓▓ \▓▓▓▓▓▓ \▓▓▓▓▓▓ \▓▓ \▓▓\▓▓▓▓▓▓▓ \▓▓▓▓▓▓ \▓▓ \▓▓▓▓▓▓▓

2023-09-04 15:39:23 I/main: Backend: version 2.16.0-RC, built on 2023-09-03 23:55:17.
2023-09-04 15:39:23 I/main: Frontend Terminal: version 2.16.0-RC, provided by Mamoe Technologies
2023-09-04 15:39:23 I/main: Welcome to visit https://mirai.mamoe.net/
2023-09-04 15:39:24 I/plugin: Successfully loaded plugin Chat Command v0.5.0
2023-09-04 15:39:24 I/plugin: Successfully loaded plugin tiktok-link-resolver v0.1.0
2023-09-04 15:39:24 I/QRLogin: Loading QRLogin v0.2.0
2023-09-04 15:39:24 W/QRLogin: 本插件仅在 2.15.0 或以上工作，且在正式版或先行发布版(RC)中可能无法使用
2023-09-04 15:39:24 I/plugin: Successfully loaded plugin QRLogin v0.2.0
2023-09-04 15:39:24 I/fix-protocol-version: 协议版本检查更新...
2023-09-04 15:39:24 I/fix-protocol-version: ANDROID_PHONE load from file:///root/content/apps/back/mcl/android_phone.json
2023-09-04 15:39:25 I/fix-protocol-version: ANDROID_PAD load from file:///root/content/apps/back/mcl/android_pad.json
2023-09-04 15:39:25 I/fix-protocol-version: 注册服务...
2023-09-04 15:39:25 I/fix-protocol-version: 服务配置文件:
file:///root/content/apps/back/mcl/KFCFactory.json
2023-09-04 15:39:25 I/plugin: Successfully loaded plugin fix-protocol-version v1.9.11
2023-09-04 15:39:25 I/plugin: Cannot load plugin mirai-api-http v2.9.1
2023-09-04 15:39:25 E/main: Failed to init MiraiConsole.
net.mamoe.mirai.console.plugin.loader.PluginLoadException: Exception while loading mirai-api-http v2.9.1
at net.mamoe.mirai.console.internal.plugin.BuiltInJvmPluginLoaderImpl.load(BuiltInJvmPluginLoaderImpl.kt:373)
at net.mamoe.mirai.console.internal.plugin.BuiltInJvmPluginLoaderImpl.load(BuiltInJvmPluginLoaderImpl.kt:40)
at net.mamoe.mirai.console.plugin.jvm.JvmPluginLoader$BuiltIn.load(JvmPluginLoader.kt)
at net.mamoe.mirai.console.plugin.jvm.JvmPluginLoader$BuiltIn.load(JvmPluginLoader.kt:54)
at net.mamoe.mirai.console.internal.plugin.PluginManagerImpl.loadPluginNoEnable(PluginManagerImpl.kt:100)
at net.mamoe.mirai.console.internal.plugin.PluginManagerImpl.loadAllPluginsUsingBuiltInLoaders$mirai_console(PluginManagerImpl.kt:164)
at net.mamoe.mirai.console.internal.MiraiConsoleImplementationBridge.doStart$mirai_console(MiraiConsoleImplementationBridge.kt:303)
at net.mamoe.mirai.console.MiraiConsoleImplementation$Companion.start(MiraiConsoleImplementation.kt:512)
at net.mamoe.mirai.console.terminal.MiraiConsoleTerminalLoader.startAsDaemon(MiraiConsoleTerminalLoader.kt:182)
at net.mamoe.mirai.console.terminal.MiraiConsoleTerminalLoader.startAsDaemon$default(MiraiConsoleTerminalLoader.kt:181)
at net.mamoe.mirai.console.terminal.MiraiConsoleTerminalLoader.main(MiraiConsoleTerminalLoader.kt:59)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.itxtech.mcl.Utility.bootJars(Utility.java:86)
at org.itxtech.mcl.Utility.bootJars(Utility.java:76)
at org.itxtech.mcl.Utility.bootMirai(Utility.java:98)
at org.itxtech.mcl.module.builtin.Boot.boot(Boot.java:113)
at org.itxtech.mcl.module.ModuleManager.phaseBoot(ModuleManager.java:123)
at org.itxtech.mcl.Loader.lambda$start$4(Loader.java:196)
at org.itxtech.mcl.Loader.tryCatching(Loader.java:151)
at org.itxtech.mcl.Loader.start(Loader.java:196)
at org.itxtech.mcl.Loader.main(Loader.java:84)
Caused by: java.lang.IllegalStateException: A JvmPlugin loaded by plugin.yml must has same description reference
at net.mamoe.mirai.console.internal.plugin.BuiltInJvmPluginLoaderImpl.load(BuiltInJvmPluginLoaderImpl.kt:362)
... 23 more`

--- a/mirai-console/backend/mirai-console/src/plugin/jvm/JvmPluginDescription.kt
+++ b/mirai-console/backend/mirai-console/src/plugin/jvm/JvmPluginDescription.kt
@@ -16,6 +16,7 @@ import io.github.karlatemp.caller.StackFrame
 import kotlinx.serialization.Serializable
 import net.mamoe.mirai.console.compiler.common.ResolveContext
 import net.mamoe.mirai.console.compiler.common.ResolveContext.Kind.*
+import net.mamoe.mirai.console.internal.plugin.JvmPluginClassLoaderN
 import net.mamoe.mirai.console.internal.util.findLoader
 import net.mamoe.mirai.console.plugin.description.PluginDependency
 import net.mamoe.mirai.console.plugin.description.PluginDescription
@@ -47,6 +48,11 @@ public interface JvmPluginDescription : PluginDescription {
             pluginClassloader: ClassLoader = CallerFinder.getCaller()?.findLoader()
                 ?: error("Cannot find caller classloader, please specify manually."),
         ): JvmPluginDescription {
+            if (filename == "plugin.yml") {
+                val cache = (pluginClassloader as? JvmPluginClassLoaderN)?.pluginDescriptionFromPluginResource
+                if (cache != null) return cache
+            }
+
             val stream = pluginClassloader.getResourceAsStream(filename)
                 ?: error("Cannot find plugin description resource '$filename'")
 

